Remove custom CSS

brianeroth opened the pull request:
This PR removes as much custom CSS in favor of Bootstrap CSS and components as possible

brianeroth committed:
Initial commit

brianeroth committed:
More

brianeroth committed:
Code formatting

brianeroth committed:
More

dlobron commented:
This looks great, Brian!  There are a lot of differences that I think are just formatting, so it's a little hard to see just the changes.  I see you replaced the appnitro CSS with bootstrap, which is excellent- that's the main thing I was hoping to do.  Can you just confirm that your changes were to the CSS?  I am testing it out now, and it really looks awesome.  Did you have a chance to try it on a mobile device?  I suspect it will be a lot better.

I got an access error when I tested, which I fixed by changing the root DB password from "password" to "root" (on the actual site, of course, neither of those is the password).  Can you just confirm that you changed it?  Either default is fine- just want to make sure.

brianeroth committed:
Revert constants

brianeroth committed:
Revert unintended changes

brianeroth commented:
> This looks great, Brian! There are a lot of differences that I think are just formatting, so it's a little hard to see just the changes. I see you replaced the appnitro CSS with bootstrap, which is excellent- that's the main thing I was hoping to do. Can you just confirm that your changes were to the CSS? I am testing it out now, and it really looks awesome. Did you have a chance to try it on a mobile device? I suspect it will be a lot better.
> 
> I got an access error when I tested, which I fixed by changing the root DB password from "password" to "root" (on the actual site, of course, neither of those is the password). Can you just confirm that you changed it? Either default is fine- just want to make sure.

@dlobron,

Yep, I ran the PHP files through a formatter to make things a little easier to read. 

The actual code changes I made are all related to CSS/HTML markup, no functionality changes. I tried it on a smaller device and things look relatively good in terms of responsiveness - there's still some weirdness that happens because of the tooltips, however. 

I accidentally checked in the `constants.php` changes I had to make to get things running, but I've reverted that change now.

If this looks good to you, what do you think about getting this PR merged in to avoid the diff getting too large, and then I'll continue on removing any more custom CSS we can?

brianeroth committed:
Bugfix

brianeroth committed:
More buttons

brianeroth committed:
Remove unused

brianeroth committed:
Remove more

brianeroth committed:
Use a well

dlobron reviewed (approved):
Looks good!
